cmd/batch-handlers.go

```diff
@@ -381,6 +383,12 @@ func (r *BatchJobReplicateV1) StartFromSource(ctx context.Context, api ObjectLay
 		skipReplicate := false
 
 		for obj := range objInfoCh {
+			if s3Type && obj.StorageClass == s3StorageClassGlacier {
```

Comment:
It doesn't look s3Type is strictly needed

Reply:
It is added on purpose if GLACIER tier name was used in MinIO's case ? :P